pyproject.toml

```diff
@@ -58,18 +58,13 @@ testpaths = [
 [tool.pylint]
     [tool.pylint.'MESSAGES CONTROL']
     disable = [
-        # https://black.readthedocs.io/en/stable/compatible_configs.html
-        "C0330",
```

Comment:
When upgrading Pylint, it emitted warnings that these config options are no longer supported